### ISSUE TITLE
Open a discussion channel on Slack/Discord/etc to improve project life

### ISSUE THREAD

**Issue by lwouis** [lwouis/alt-tab-macos#79]
Issues are nice to discuss specific topics but i think this project would benefit from having a chat where people can discuss more casually.

Services like Discord or Gitter have free hosting.

This ticket is about picking a chat provider (evaluating best practice in the OSS community here), opening a chat room there, and adding that info in the readme.

**Comment by gingerr** [comment]
Great idea. I only know Discord (for gaming), Slack (work), MS Teams (work) and Mattermost (work, but discontinued).

Gitter is looking good.

**Comment by mfn** [comment]
Gitter is probably the most easiest when Github collaboration is already "present" (though I also use Slack for some OSS projects and it's also fine).

**Comment by lwouis** [comment]
I went with Discord for the moment. I tried Gitter since it's integrated nicely with Github and I liked that. However I face major bugs on their service while trying to create the server, link to it, integrate with github and travis, etc. It was really difficult to workaround their issues. Cleaning cookies/everything didn't help so I had to proceed in incognito mode. Pretty bad experience so I tried Discord and had an straightforward experience there. There are way more features also on Discord.

I'll close for now. Feel free to bring more arguments into another ticket to discuss either adding another server elsewhere, or a migration. I'm open to suggestion but I think let's try Discord first so we have *something* available fast
